omnigibson/robots/r1.py

```diff
@@ -187,15 +187,15 @@ def assisted_grasp_start_points(self):
     def assisted_grasp_end_points(self):
         return {
             arm: [
-                GraspingPoint(link_name=f"{arm}_gripper_link1", position=th.tensor([-0.032, 0.0, -0.009])),
-                GraspingPoint(link_name=f"{arm}_gripper_link1", position=th.tensor([0.025, 0.0, -0.009])),
+                GraspingPoint(link_name=f"{arm}_gripper_link2", position=th.tensor([-0.032, 0.0, -0.01])),
+                GraspingPoint(link_name=f"{arm}_gripper_link2", position=th.tensor([0.025, 0.0, -0.01])),
             ]
             for arm in self.arm_names
         }
 
     @property
-    def base_link_names(self):
-        return ["base_link"]  # , "wheel_link1", "wheel_link2", "wheel_link3"]
+    def floor_touching_base_link_names(self):
```

Comment:
need to add `non_floor_touching_base_link_names` to be `["base_link"]`

Reply:
I set `non_floor_touching_base_link_names` to `[self.base_footprint_link_name]` by default, which is fairly reasonable I think.